modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/UserAccountResourceImpl.java

```diff
@@ -651,7 +652,7 @@ public UserAccount postAccountUserAccount(
 			_accountEntryUserRelService.addAccountEntryUserRel(
 				accountId, contextUser.getUserId(),
 				userAccount.getAlternateName(), userAccount.getEmailAddress(),
-				contextAcceptLanguage.getPreferredLocale(),
+				LocaleUtil.fromLanguageId(userAccount.getLanguageId()),
```

Comment:
Can we use the old behavior as a fallback? If userAccount.getLanguageId is null use the contextAcceptLanguage.getPreferredLocale()

Reply:
See: 3aa7c8e473cb3325b6828222eb60d3d50f7fec2b